.github/CODEOWNERS

```diff
@@ -42,26 +46,37 @@
 # PRLabel: %AgriFood
 /sdk/agrifood/azure-agrifood-farming/                                @bhargav-kansagara
 
+# AzureSdkOwners: @xiangyan99
+# ServiceLabel: %Azure.Identity
 # PRLabel: %Azure.Identity
 /sdk/identity/                                                       @pvaneck @xiangyan99 @Azure/azure-sdk-write-identity
 
+# AzureSdkOwners: @kashifkhan
+# ServiceLabel: %Event Hubs
 # PRLabel: %Event Hubs
 /sdk/eventhub/                                                       @annatisch @kashifkhan @swathipil @l0lawrence
```

Comment:
This format will include our team as the service owners for `Event Hubs` and combine the list with the `/<NotInRepo>/` block below when `Service Attention` is aded.

Reply:
We own the library, yes, but we should include our service team contacts for when `Service Attention` is added.